README.md

```diff
@@ -1,69 +1,77 @@
 # Cross-platform normalization enables machine learning model training on microarray and RNA-seq data simultaneously
 
-The full output of a [version](https://github.com/greenelab/RNAseq_titration_results/releases/tag/v1.1) of this analysis is available at Figshare under the DOI: [10.6084/m9.figshare.5035997.v2](https://doi.org/10.6084/m9.figshare.5035997.v2)
-
 ## Summary
 
 We performed a series of supervised and unsupervised machine learning 
-evaluations, as well as differential expression analyses, to assess which 
+evaluations, as well as differential expression and pathway analyses, to assess which 
 normalization methods are best suited for combining data from microarray and 
 RNA-seq platforms. 
 
-We evaluated five normalization approaches for all methods: 
+We evaluated six normalization approaches for all methods: 
 
 1. log-transformation (LOG) 
 2. [non-paranormal transformation](https://arxiv.org/abs/0903.0649) (NPN)
 3. [quantile normalization](http://bmbolstad.com/misc/normalize/bolstad_norm_paper.pdf) (QN)
-4. [Training Distribution Matching](https://peerj.com/articles/1621/) (TDM)
-5. standardizing scores (z-scoring; Z).
-
-A [version](https://github.com/greenelab/RNAseq_titration_results/releases/tag/v1.0) of this project is detailed in our pre-print
-[Cross-Platform Normalization Enables Machine Learning Model Training On Microarray And RNA-Seq Data Simultaneously](https://doi.org/10.1101/118349).
-
-_We are actively making improvements to this codebase; see [#12](https://github.com/greenelab/RNAseq_titration_results/issues/12)._
- 
-## Breast Cancer Data
+4. quantile normalization followed by z-scoring (QN-Z)
+5. [Training Distribution Matching](https://peerj.com/articles/1621/) (TDM)
+6. z-scoring (Z)
 
-[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.58862.svg)](https://doi.org/10.5281/zenodo.58862)
+## Download data from The Cancer Genome Atlas (TCGA)
 
-The Cancer Genome Atlas BRCA data used for these analyses
+TCGA data from 520 breast cancer (BRCA) patients used for these analyses
 is [available at zenodo](https://zenodo.org/record/58862).
 
+Data from 150 glioblastoma (GBM) patients is available from the [Genomic Data Commons PanCan Atlas](https://gdc.cancer.gov/about-data/publications/pancanatlas).
+
 ```
-# To download data, run in top directory:
-sh brca_data_download.sh
+# To download data, run the data download script in the top directory:
+bash download_TCGA_data.sh
 ```
 
 ## Analysis
```

Comment:
If you take my suggestion about moving the recreation instructions, I might change this to:

```suggestion
## Methods
```